helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -52,6 +52,7 @@
     FAULT_ZONE_TYPE, // the type in which isolation should be applied on when Helix places the
     // replicas from same partition.
     TOPOLOGY_AWARE_ENABLED, // whether topology aware rebalance is enabled.
+    VIRTUAL_GROUP_ENABLED, // whether virtual topology group is enabled
```

Comment:
The idea of setting this is to avoid unexpected API call to a wrong cluster. We'd like to have this cluster config to be safer.

Reply:
As we discussed offline yesterday, it is a must as long as we still want to keep the physical topology somewhere and may want to provide a way to roll back.

But logic-wise, it introduced more dependencies. For example, what if I enable virtual group without enabling topology awareness, does it even makes sense? What about when cloud features are not enabled? I can foresee many questions from our users.

Reply:
As discussed offline, there is a design change and we are discarding `VIRTUAL_GROUP_ENABLED`. Rollback is not a concern because there is always a fixed topology config on cloud environment.